### PR TITLE
feat(mobile-matador): Implement Homepage Appbar

### PR DESCRIPTION
**Pull request by duressaJemal**
Implement homepage appbarr UI that contains navigation menu button, title, and profile avatar with necessary styling applied. 

screenshoot:

app_barr_implementation

### CODE REVIEW

**Review by BekyH — CHANGES_REQUESTED** [review]
Don't Modify the example folder. It is a placeholder to show the structure. So create your own folder inside the features directory with feature_name and make it a similar structure to the example folder. Migrate all the changes made in the example folder.

File: Mobile/starter_project_matador/lib/features/home_page/presentation/widgets/appbar_title.dart [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+class AppBarTitle extends StatelessWidget {
+  const AppBarTitle({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Text(
+        "Welcome Back!",
+        style: GoogleFonts.poppins(
```
Comment (lealemgetahun) [inline Mobile/starter_project_matador/lib/features/home_page/presentation/widgets/appbar_title.dart:14]: Don't use google fonts use asset for font

**Review by lealemgetahun — APPROVED** [review]

**Review by lealemgetahun — APPROVED** [review]

**Review by BekyH — DISMISSED** [review]

**Review by lealemgetahun — APPROVED** [review]

**Review by lealemgetahun — DISMISSED** [review]

**Review by Adona12 — APPROVED** [review]
LGTM

**Review by Redi22 — CHANGES_REQUESTED** [review]

**Review by Redi22 — APPROVED** [review]
LGTM

**Review by YiseBoge — APPROVED** [review]
let's avoid committing everything in the future

**Pull request merged**